channels/chan_simpleusb.c

```diff
@@ -906,6 +908,35 @@ static void *hidthread(void *arg)
 		 * found device.
 		 */
 		ast_radio_time(&o->lasthidtime);
+		/* If configuration has a serial number defined, find the device */
+		if (strlen(o->serial) > 0)
+		{
+			ast_log(LOG_NOTICE, "Checking for USB device with serial %s\n", o->serial);
```

Comment:
This ast_log() ... and the others below ... really feel like debug messages.  Maybe switch them to ast_debug() ?